### PR TITLE
#6345 Change on notes Author size and color

### LINKED ISSUE #6345: Add # of comments and date to post cards
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://code.publiclab.org/#r=all), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @publiclab/reviewers to get it labelled. Or, if this is not your first time, try to solve it yourself!

****

We need some small tweaks to the post cards shown here (building on ideas from https://github.com/publiclab/plots2/issues/6307):

image

These appear on pages like https://publiclab.org/tag/babylegs

It would be great to get them looking more like this:

image

This would include:

1. making the "Post by ______" smaller and grey (`#222` is a good color)
2. making it closer to the title, maybe by deleting the empty "meta" paragraph tag
3. adding a new line below it, starting with the "comment" icon from [FontAwesome](https://fontawesome.com/v4.7.0/icon/comment-o): `<i class="fa fa-comment-o"></i> 1` in a link, with the # of comments (`<%= node.comments.size %>`)
4. making the display of time more minimal as shown in the mockup above

Here is the code for the card:

https://github.com/publiclab/plots2/blob/7e2ebf018d021f6f31d5a9d99d242eb5048ba2ba/app/views/notes/_notes.html.erb#L34-L44

We'd love help with this, thanks!!!

### PR DESCRIPTION
**Pull request by SergiOca87**
Fixes #6345

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by CoderJolly** [comment]
@SergiOca87 It would be great if you could reference the issue number in the PR (Fixes #<issue number>)

File: app/views/notes/_notes.html.erb [hunk 1]
```diff
@@ -104,6 +104,16 @@
   text-align: left;
 }
 
+#notes .card-body small{
+    font-size: 15px;
```
Comment (mathyoudawson) [inline app/views/notes/_notes.html.erb]: Did you consider adding another class to #notes

**Comment by jywarren** [comment]
This is the latest render from the code! 

image

Compare to previously:

image

and to the goal:

**Comment by jywarren** [comment]
I wonder if we could get closer in size to the goal, and make the lower text grey? Thank you for your help!

**Comment by cesswairimu** [comment]
Hello, I am sorry but it seems like this was solved in another PR. Closing this...Let me know if you need help finding another issue. Thanks

**Pull request closed without merging**